recipe/meta.yaml

```diff
@@ -7,41 +7,46 @@ package:
 source:
   fn: ffmpeg-{{ version }}.tar.gz  # [not win]
   url: https://ffmpeg.org/releases/ffmpeg-{{ version }}.tar.gz  # [not win]
-  sha256: ecb99b0d191c6a16d114d2ee4b91e4ca824f67d31565dd6de9d1c26ee192739f  # [not win]
+  sha256: dc4b1c622baa34fc68d763cd2818e419d1af90271e0506604905f25a46ea8273  # [not win]
+  patches:  # [osx]
+    - patch-libavcodec-videotoolboxenc.c-drop-Nullable.patch  # [osx]
```

Comment:
Would be nice to have a comment above the patch explaining its purpose (e.g. "working around XCode 6 issues" or similar). Also would add the ref below in that comment so people can read about the patch if they are curious why it's needed. :)

ref: https://trac.macports.org/ticket/55746